.gitignore

```diff
@@ -58,3 +58,4 @@ HOWTO-web2py-devel
 *.sublime-project
 *.sublime-workspace
 .idea/*
+*.DS_Store
```

Comment:
you're missing a new line at the end of file (which you can see if you roll the mouse over)